front/components/agent_builder/AgentBuilderContext.tsx

```diff
@@ -1,8 +1,7 @@
 import { createContext, useContext, useState } from "react";
 import { useEffect } from "react";
 
-import { mcpServerViewSortingFn } from "@app/lib/actions/mcp_helper";
-import type { MCPServerViewType } from "@app/lib/api/mcp";
+import { MCPServerViewsProvider } from "@app/components/assistant_builder/contexts/MCPServerViewsContext";
```

Comment:
Once I finish moving all fetch to the client side, I will create a duplicate component for agent_builder, until then I will reuse the one from assistant_builder so I don't have to fix both if there is an issue